src/binder/bind/bind_file_scan.cpp

```diff
@@ -24,6 +24,10 @@ namespace binder {
 FileTypeInfo bindSingleFileType(main::ClientContext* context, const std::string& filePath) {
     std::filesystem::path fileName(filePath);
     auto extension = context->getVFSUnsafe()->getFileExtension(fileName);
+    if (extension.empty()) {
```

Comment:
I think the extension name might be empty if you are scanning from the delta/iceberg. So we shouldn't blindly error when the extension is an empty string.
Can you check how duckdb handle this?

Reply:
let me check this

Reply:
Fixed this